test/unit/offset.js

```diff
@@ -75,10 +75,11 @@ QUnit.test( "disconnected element", function( assert ) {
 	assert.equal( result.top, 0, "Retrieving offset on disconnected elements returns zeros (gh-2310)" );
 	assert.equal( result.left, 0, "Retrieving offset on disconnected elements returns zeros (gh-2310)" );
 	assert.equal( Object.keys( result ).length, 2, "Retrieving offset on disconnected elements returns offset object (gh-3167)" );
+	assert.equal( jQuery.isPlainObject( result ), true, "Retrieving offset on disconnected elements returns plain object (gh-3612)" );
```

Comment:
Should we be actually trying to mutate the object here? The `jQuery.isPlainObject()` test is a bit of an indirect feature test and technically we could be returning a plain object with read-only properties.